stdlib/universe/pivot.go

```diff
@@ -424,103 +405,150 @@ func (t *pivotTransformation) Finish(id execute.DatasetID, err error) {
 	t.d.Finish(err)
 }
 
-// pivotTransformation2 is an optimized version of pivot.
+type SortedPivotProcedureSpec struct {
+	plan.DefaultCost
+	RowKey      []string
+	ColumnKey   []string
+	ValueColumn string
+}
+
+func (s *SortedPivotProcedureSpec) Kind() plan.ProcedureKind {
+	return PivotKind
+}
+func (s *SortedPivotProcedureSpec) Copy() plan.ProcedureSpec {
+	ns := new(SortedPivotProcedureSpec)
+	ns.RowKey = make([]string, len(s.RowKey))
+	copy(ns.RowKey, s.RowKey)
+	ns.ColumnKey = make([]string, len(s.ColumnKey))
+	copy(ns.ColumnKey, s.ColumnKey)
+	ns.ValueColumn = s.ValueColumn
+	return ns
+}
+
+// sortedPivotTransformation is an optimized version of pivot.
```

Comment:
Yes, `sortedPivotTransformation` is a much better name that `pivotTransformation2`. 😜

Reply:
> This looks good. Just a single minor suggestion about some comments in the code, rather than in a PR.

Yep, here's the comment that you asked for, https://github.com/influxdata/flux/blob/40fe5f33680916c66444120b92a91d748089b760/stdlib/universe/pivot_test.go#L939